host-controller/src/main/java/org/jboss/as/domain/controller/operations/HostProcessReloadHandler.java

```diff
@@ -76,10 +76,13 @@
             .setAlternatives(ModelDescriptionConstants.USE_CURRENT_DOMAIN_CONFIG)
             .build();
 
+    protected static final AttributeDefinition TIMEOUT = new SimpleAttributeDefinitionBuilder(ModelDescriptionConstants.TIMEOUT, ModelType.INT, true)
+            .setDefaultValue(new ModelNode(0)).build();
```

Comment:
I need to triple check but I don't believe the suspend op blocks waiting for the server to suspend. Actually, looking at the ServerSuspendHandler code I suspect it's quite broken. Hope I'm wrong though.

Looking how ServerSuspendHandler is written though I think the intent was to have some sort of blocking behavior, but I don't think that's what really happens.